docs/cli/run.md

```diff
@@ -60,10 +60,16 @@ $ bun index.js
 
 To run a file in watch mode, use the `--watch` flag.
 
+> `--watch` won't work if you place it anywhere in the command other than directly after 'bun'.
```

Comment:
Thank you for this

It should be a notice instead 

People might not read it closely and copy paste the wrong command

Reply:
Done.